src/EditorFeatures/Core/Implementation/Formatting/FormatCommandHandler.FormatDocument.cs

```diff
@@ -43,15 +43,11 @@ private bool TryExecuteCommand(FormatDocumentCommandArgs args)
             }
 
             var result = false;
-            _waitIndicator.Wait(
-                title: EditorFeaturesResources.Format_Document,
-                message: EditorFeaturesResources.Formatting_document,
-                allowCancel: true,
-                action: waitContext =>
-                {
-                    Format(args.TextView, document, null, waitContext.CancellationToken);
-                    result = true;
-                });
+            using (context.WaitContext.AddScope(allowCancellation: true, EditorFeaturesResources.Formatting_document))
+            {
+                Format(args.TextView, document, null, context.WaitContext.CancellationToken);
+                result = true;
```

Comment:
Behavior changed here: previously _waitIndicator.Wait would have caught the cancelation being thrown, which is why result could end up false. Now the pattern doesn't make sense.